VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/NotificationsSettingsScreen/NotificationsSettingsScreen.tsx

```diff
@@ -110,14 +110,14 @@ function NotificationsSettingsScreen({ navigation }: NotificationsSettingsScreen
               {preferenceList()}
             </>
           ) : (
-            <AlertBox
-              border={'informational'}
-              title={t('notifications.settings.alert.title')}
-              text={t('notifications.settings.alert.text')}>
+            <AlertWithHaptics
+              variant="info"
+              header={t('notifications.settings.alert.title')}
+              description={t('notifications.settings.alert.text')}>
               <Box mt={standardMarginBetween}>
                 <Button onPress={goToSettings} label={t('notifications.settings.alert.openSettings')} />
```

Comment:
Could potentially be simplified moving this to the `primaryButton` or `secondaryButton` props to AlertWithHaptics.